README.md

```diff
@@ -40,13 +40,32 @@ Flux is most useful when used as a deployment tool at the end of a
 Continuous Delivery pipeline. Flux will make sure that your new
 container images and config changes are propagated to the cluster.
 
-Among its features are:
+### Features
+
+Its major features are:
 
 - [Automated git → cluster synchronisation](/site/introduction.md#automated-git-cluster-synchronisation)
 - [Automated deployment of new container images](/site/introduction.md#automated-deployment-of-new-container-images)
-- [Integrations with other devops tools](/site/introduction.md#integrations-with-other-devops-tools) ([Helm](site/helm/helm-integration.md) and more)
+- [Integrations with other devops tools](/site/introduction.md#integrations-with-other-devops-tools) ([Helm](/site/helm/helm-integration.md) and more)
 - No additional service or infrastructure needed - Flux lives inside your
   cluster
+- Straight-forward control over the state of deployments in the
+  cluster (rollbacks, lock a specific version of a workload, manual
+  deplomyents, )
+- Observability: Audit trail (who did what when) and leave messages as
```

Comment:
"git commits are an audit trail, and you can record e.g., why a given deployment was locked.